proseco/guide.py

```diff
@@ -39,6 +42,8 @@ def get_guide_catalog(obsid=0, **kwargs):
 
     :returns: GuideTable of acquisition stars
     """
+    global STAR_PAIR_DIST_CACHE
```

Comment:
@taldcroft I'm not sure about the most reasonable place to clear the cache of star-pair distances.  I think this code would only be called via get_guide catalog.

Reply:
Here is fine.  Not sure why you are declaring the var to be global here, that is not required for modifying this dict.

Plan B for more "correctness" (at the cost of a little "messy") would be creating a `dist_cache` var in `select_catalog` and then passing it down through the layers.  I could go either way.

Reply:
For now, just removing the global is good enough.